nexpose/nexpose_site.py

```diff
@@ -176,6 +180,38 @@ def AsXML(self, exclude_id):
         xml_scanconfig.append(xml_scheduling)
         xml_data.append(xml_scanconfig)
 
+        xml_tags = create_element('Tags')
+        for tag in self.tags:
+            for tag_data in tag:
+                tag_data_name = get_attribute(tag_data, 'name')
+                tag_data_value = get_attribute(tag_data, 'value')
```

Comment:
Per flake8 output, `tag_data_value` is never used. Should probably be used in place of the calls to `get_attribute(tag_data, 'value')` in the below `if` blocks since the value shouldn't change until the name matches a condition.